packages/gatsby-theme-docs/bin/create-release-note.js

```diff
@@ -35,36 +36,44 @@ const questions = [
     ],
   },
   {
-    type: 'date',
+    type: 'text',
     name: 'date',
-    message: 'Pick a release date',
+    message: 'Insert the release date with the following format: 2021-01-21',
+    validate: (date) => {
+      const checkDateFormat = moment(date, 'YYYY-MM-DD', true);
+      return checkDateFormat.isValid() ? true : 'Please use a valid format!';
+    },
   },
   {
     type: 'text',
     name: 'title',
-    message: 'The title of the release note. Max 256 character plain text.',
+    message: 'Insert the title for the release note. Write it in plain text.',
     validate: (title) =>
       title.length > 256 ? 'Title is longer than 256 chars!' : true,
   },
   {
     type: 'text',
     name: 'description',
     message:
-      'Key changes and their advantages. Max 256 character plain text. This text is tweet-able and used in RSS feeds.',
-    format: (val, values) => `${values.title} ${val}`,
+      'Describe the key changes and their advantages briefly. Max 256 character plain text. This text is tweet-able and used in RSS feeds.',
     validate: (description) =>
       description.length > 256 ? 'Description is longer than 256 chars!' : true,
   },
   {
     type: 'list',
     name: 'topics',
     message:
-      'Write down the topics for this release note. Seperate them with commas.',
+      'Fill in the topics for this release note. Seperate them with commas.',
     separator: ',',
   },
 ];
 
 (async () => {
+  if (!fs.existsSync(path.join(process.cwd(), `src/releases`))) {
```

Comment:
Nit: Declare the releases folder path as a variable since you also use it when writing the file.